src/utils/createDynamicAppAtScriptTag.ts

```diff
@@ -23,21 +24,30 @@ export const createDynamicAppAtScriptTag = async (
 ): Promise<void> => {
   const { elementID, lang } = config
 
+  const scriptEl = getCurrentScript()
+  const props = getAllScriptURLParameters(scriptEl)
+  const merchantId = props.merchantId
+  const theme = merchantId
```

Comment:
I think `merchantId` is supposed to be a uuid, not a representation of the theme itself. I don't think we're clearly separating concerns if we allow `merchantId` to both be a unique identifier for the merchant as well as a way to set the default theme. Please make `theme` a separate parameter returned from `getAllScriptURLParameters`